docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc

```diff
@@ -57,13 +57,17 @@ For each repository previously cloned, run `npm install` so regular NPM dependen
 [discrete]
 == Starting the Core Server
 
+Run the `npm run start` command to start the Core server as follows:
+
 [source,bash]
 ----
 cd raincatcher-server/
-npm run start:ts
+npm run start
```

Comment:
This in fact is ok for now, but after release this will change. 
ping @paolobueno to revert that in 1.1.0 version.

I need to get that merged and revert website to not cause issues for newcomers.